docs/studio: top-level plots (#4121)
iterative/dvc.org: content/docs/studio/user-guide/projects-and-experiments/visualize-and-compare.md

2. Image files in JPEG, GIF, PNG, or SVG format. These images will be displayed
   directly in Iterative Studio.

> You can define multiple plots in a single repository. The plots can be
> [stage-specific](/doc/user-guide/experiment-management/visualizing-plots#plot-outputs)
> or
> [top-level](/doc/user-guide/experiment-management/visualizing-plots#top-level-plots).
> Below is an example snippet from a `dvc.yaml` file showing the `evaluate` stage
> of the DVC pipeline, with 2 different plots. It also includes a top-level plot
> which uses data from the file `runtime_logs/logs.csv`.

Reviewer: ```suggestion
You can define plots in one or more `dvc.yaml` files. These can either be
[pipeline outputs](/doc/user-guide/experiment-management/visualizing-plots#plot-outputs)
or
[top-level](/doc/user-guide/experiment-management/visualizing-plots#top-level-plots).
definitions. Below is a sample `dvc.yaml` file with a plot in the
`evaluate` stage and aanother at the top-level:
```

Reviewer 2: @jorgeorpinel I've incorporated the changes, but with some modification. PTAL.

Reviewer: Thanks

```yaml
stages:

>   evaluate:
>     cmd: python src/evaluate.py
>     deps:
>       - output/data.pkl
>       - output/model.h5
>       - src/evaluate.py
>     metrics:
>       - output/metrics.json:
>           cache: false
>     plots:
>       - output/predictions.json:
>           cache: false
>           template: confusion
>           x: actual
>           y: predicted
>       - output/misclassified_samples/:
>           cache: false
>   ...
>   ...
> plots: runtime_logs/logs.csv

Reviewer: ```suggestion
  evaluate:
    cmd: python src/evaluate.py
    deps: ...
    plots:
      - output/predictions.json: 
          template: confusion
          x: actual
          y: predicted

plots: output/misclassified_samples/ 
```

☝🏼 The top-level example is somewhat plain though, not really showing the benefit of flexible plots.

Reviewer 2: @jorgeorpinel I've incorporated the changes, but with some modification. PTAL. Specifically, I didn't move the image plots to top-level, because `misclassified_samples` are usually the output of model evaluation.

Reviewer: OK

```

> As you can see,
> 
> - metrics from `output/predictions.json` will be rendered in a confusion matrix,
> - images in the `output/misclassified_samples/` directory will be displayed

Reviewer: ```suggestion
1. Metrics from `output/predictions.json` will be plotted in a confusion matrix.
```

Reviewer 2: Updated. PTAL.

>   directly,
> - data from `runtime_logs/logs.json` will be rendered using the default (linear)
>   template.

Reviewer: ```suggestion
2. Images in `output/misclassified_samples` will be displayed directly (a single
   image file could be used instead).
```

Reviewer 2: @jorgeorpinel I didn't make this change as it's not clear `a single
   image file could be used instead "of what?"`.

Reviewer: Instead of "Images in `output/misclassified_samples`". Could also rephrase that to "in the `output/misclassified_samples` directory", if that's clearer.

> For images, you can also specify a single image file (eg,
> `output/misclassified_sample1.png`) instead of a directory.
> 

Reviewer: ```suggestion
```

Reviewer 2: Didn't change this. Pls see [above comment](https://github.com/iterative/dvc.org/pull/4121#discussion_r1034324349).
